infra/notifications/src/app.js

```diff
@@ -50,6 +50,11 @@ process.argv.forEach(arg => {
 })
 
 const config = {
+  // URL for messaging on dapp
+  dappMessagesUrl:
+    args['--dapp-messages-url'] ||
+    process.env.DAPP_MESSAGES_URL ||
+    'https://dapp.originprotocol.com/#/messages',
```

Comment:
Do we want this default value ? Is there a risk it could cause messages to be sent to production when developers test on their local environment ?

Reply:
Maybe not for this PR but we have DAPP_OFFER_URL and DAPP_MESSAGES_URL, all we really need is DAPP_URL because the paths can just be added as they don't change. And we already have all the DAPP_URLS for each network defined in emailSend.js.

We could just set a NETWORK_ID env var and then not have to parse that out of the listing in emailSend.js

@franck this is the URL that'll open when you click on the push notification, so the worst case is that this will cause someone to open messages on the wrong network on mobile.

Reply:
Love the idea of a `DAPP_URL` and/or `NETWORK_ID` var. It's a lot of gyrations to get it from the listing, and for messaging there is no clue what network we're on. 

(Frank, that URL was originally hard-coded..so at least it was moving in the right direction. :)

Reply:
Changed to just `ETH_NETWORK_ID`, and all other URLs are derived from that. _Much_ cleaner.

Fix is in: 96ff1fa62779ed4a0cc94e4032501f371dd25368